main.go

```diff
@@ -67,8 +68,46 @@ func usage() {
 }
 
 func main() {
+	log.SetFlags(0)
+	log.SetPrefix("")
+
 	flag.Usage = usage
 	flag.Parse()
+
+	// do some basic validation on the outfile format
+	// indexing is safe because we're searching for an ascii char
```

Comment:
I think your rules are reasonable, I just don't like the implementation of the checker! The general case of checking fmt strings is hard.

What do you think if we left out the checking completely and trusted the user?  After all the users are technical users. Or maybe just made a warning if `!strings.Contains("%v")`?